Lib/unittest/async_case.py

```diff
@@ -116,6 +116,8 @@ def _setupAsyncioRunner(self):
         assert self._asyncioRunner is None, 'asyncio runner is already initialized'
         runner = asyncio.Runner(debug=True)
         self._asyncioRunner = runner
+        # Force loop to be initialized
```

Comment:
It would be nice to add more detailed explanation why is it needed.

Reply:
Added explanation